pkgs/development/compilers/llvm/15/llvm/default.nix

```
@@ -58,7 +58,11 @@ in stdenv.mkDerivation (rec {
 
   propagatedBuildInputs = [ ncurses zlib ];
 
-  checkInputs = [ which ];
+  checkInputs = [
+    which
+    python3.pkgs.psutil # !!! lit still doesn't seem to find this; leads to some
```

You probably want this:

```suggestion
(python3.withPackages (p: with p; [ psutil ]))
```

---

oh whoops; I thought the `python3` setup hook handled adding this to the python path

I'll give this a go, thanks